src/i18n/strings/en_US.json

```diff
@@ -199,6 +199,11 @@
     "You have successfully set a password and an email address!": "You have successfully set a password and an email address!",
     "Remember, you can always set an email address in user settings if you change your mind.": "Remember, you can always set an email address in user settings if you change your mind.",
     "Warning": "Warning",
+    "Pin Message": "Pin Message",
+    "Unpin Message": "Unpin Message",
+    "Jump to message": "Jump to message",
+    "No pinned messages.": "No pinned messages.",
+    "Loading...": "Loading...",
```

Comment:
Ftr these will just fall back to UK English if they're not there so adding them here doesn't actually do anything

Reply:
The only reason I include these is to make weblate not complain about en-us being <100%.